### DISCUSSION TITLE
✅ 🔗 2 ideas: Overflow scroll & infinite scrolling for products. (Duplicate)

### DISCUSSION THREAD

**Discussion by AdrianPaulV** [elementor/elementor#17980] [Feature Request]
Hey guys! Would you consider adding infinite scrolling to products?
Also I hate Divi :P but I like their 'overflow scroll' option! That'd be a great addition to manage widget, column and section overflows too and create scrollable sections.

Thanks!

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the introduction of the Loop Builder in Elementor Pro v3.9! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
